patches/server/0353-Anti-Xray.patch

```diff
@@ -1051,11 +1031,12 @@ index 568ac283baf40e2a89f5b002ffd899eba8008ef2..23d76eb74a88610472aa0288559efbaa
          this.heightmaps = new CompoundTag();
  
          for(Map.Entry<Heightmap.Types, Heightmap> entry : chunk.getHeightmaps()) {
-@@ -43,7 +46,13 @@ public class ClientboundLevelChunkPacketData {
+@@ -43,7 +46,14 @@ public class ClientboundLevelChunkPacketData {
          }
  
          this.buffer = new byte[calculateChunkSize(chunk)];
 -        extractChunkData(new FriendlyByteBuf(this.getWriteBuffer()), chunk);
++
```

Comment:
is this blank line intentional?

Reply:
Yes, I always use blank lines to separate code blocks that have indentation. This style is used in the entire patch.